content/posts/2025-09-26-Hacktoberfest-2025.adoc

```diff
@@ -0,0 +1,84 @@
+
+. *Undertow:* Undertow is a versatile and high-performance web server that provides both blocking and non-blocking APIs based on NIO. Its modular architecture allows you to build a web server tailored to your needs, from a full Java EE servlet container to a low-level non-blocking handler.
+* Here are the https://github.com/undertow-io/undertow[GitHub repository], https://github.com/undertow-io/undertow/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/issues/?jql=project%20%3D%20Undertow%20%20and%20labels%20in%20(%22good-first-issue%22)[Hacktoberfest Issues link].
+
+. *PatternFly Java:* PatternFly Java is a 💯 Java implementation of PatternFly without any JavaScript dependencies. Its goal is to provide an easy-to-use, elegant, and efficient API to build complex web applications with PatternFly in Java. PatternFly Java works with both GWT and J2CL.
+PatternFly Java is used for the next-gen WildFly management console.
+* Here are the https://github.com/patternfly-java/patternfly-java[GitHub repository], https://github.com/patternfly-java/patternfly-java/blob/main/CONTRIBUTING.md[Contributing Guide], and https://github.com/patternfly-java/patternfly-java/issues?q=is%3Aissue%20state%3Aopen%20label%3Ahacktoberfest%20label%3Agood-first-issue[Hacktoberfest Issues link].
+
+. *Narayana LRA:* Long Running Actions (abbreviated as LRA ) is a MicroProfile specification that introduces APIs for services to coordinate activities. Narayana LRA is an implementation of the specification and is included in both Quarkus and WildFly.
+* Here are the https://github.com/jbosstm/lra[GitHub repository], https://github.com/jbosstm/lra/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/issues/?jql=project%20%3D%20JBTM%20AND%20status%20in%20(Open%2C%20Reopened)%20AND%20component%20%3D%20LRA%20AND%20labels%20in%20(hacktoberfest%2C%20good-first-issue)%20ORDER%20BY%20status%20DESC[Hacktoberfest Issues link].
+
+. *Intersmash:* Intersmash is a Java library that makes it easy to automate the provisioning and execution of tests in cloud-native environments. It helps the user prototype and test complex interoperability scenarios on kubernetes compliant cloud-native environments and platforms, most notably OpenShift. (Other Kubernetes implementations will be supported in the future.)
+* Here are the https://github.com/Intersmash/intersmash[GitHub repository], https://github.com/Intersmash/intersmash/blob/main/CONTRIBUTING.md[Contributing Guide], and https://github.com/Intersmash/intersmash/issues[Hacktoberfest Issues link].
+
+. *WildFly:* https://arquillian.org/arquillian-cube/[Arquillian Cube] is an Arquillian extension that can be used to run tests on Docker containers, on Kubernetes and on OpenShift... the Arquillian way!
```

Comment:
s/WildFly/Arquillan Cube/ ?

Reply:
Thanks, while doing copy pasting I missed it.